test/styles/RRR.R

```diff
@@ -3,19 +3,23 @@
 ### Function declarations
 
 ## 1
-fun <- function(argument1,
-                argument2) {
-    body
+{
+    fun <- function(argument1,
+                    argument2) {
+        body
+    }
 }
 
 ## 2
-function(
-    argument1,
-    argument2
-    )
+{
+    function(
+             argument1,
```

Comment:
That's because `ess-indent-function-declaration` is set to `t` by default in the RRR style, so that function declarations are always indented at opening parenthesis.